tools/load-restaurants.js

```diff
@@ -0,0 +1,144 @@
+const request = require('request')
```

Comment:
nitpick - let's call the folder `scripts` instead of `tools`.

Reply:
Had it as `scripts` first and ended up renaming it 😅 changing!